lutris/game_actions.py

```diff
@@ -79,7 +79,7 @@ def on_game_stop(self, *_args):
         """Stops the game"""
         games = self.get_running_games()
         for game in games:
-            game.force_stop()
+            game.game_launcher.force_stop()
```

Comment:
Don't like this. get_running_games() should return the launchers- that is what is running, not the Game object.

This design should support running the same game twice, even if we don't want to surface that in the UI.